src/pages/ProductList/ImageSlide/ImageSlide.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import SlickDot from './SlickDot/SlickDot';
+import Slide from './Slide/Slide';
+import { SLIDE_DATA } from './SlideData';
+import './imageSlide.scss';
+
+const ImageSlide = () => {
+  const [countSlide, setCountSlide] = useState(0);
+
+  const changeSlide = slideId => {
+    setCountSlide(slideId);
+  };
+
+  return (
+    <div className="imageSlide">
+      <div className="slideList">
+        {SLIDE_DATA.filter(el => el.id === countSlide).map(el => (
+          <Slide key={el.id} slideId={el.id} imgUrl={el.imageUrl} />
+        ))}
```

Comment:
```suggestion
        {SLIDE_DATA.filter(slide => slide.id === countSlide).map(el => (
          <Slide key={el.id} slideId={el.id} imgUrl={el.imageUrl} />
        ))}
```
el과 같은 광범위한 이름은 지양해주세요, 명확히 어떤 데이터를 나타내는지 이름을 잘 지어주셔야 합니다.
이름짓기가 귀찮고 어려울수 있지만 가장 기본적이면서 가장 중요한 내용이니까 지금부터 이름을 잘 짓는 연습을 하시는게 좋습니다!

Reply:
```suggestion
        {SLIDE_DATA.filter(({id}) => id === countSlide).map(el => (
          <Slide key={el.id} slideId={el.id} imgUrl={el.imageUrl} />
        ))}
```
인자자리에서 바로 구조분해하는식으로도 적용 가능합니다!

Reply:
수정하도록 하겠습니다!